model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -174,6 +175,14 @@ case class WorkflowCostBreakdown(id: String, cost: BigDecimal, currency: String,
 
 case class ExternalEntityInfo(dataStoreId: String, rootEntityType: String)
 
+case class SubmissionOptions(useCallCache: Boolean,
```

Comment:
Assuming this doesn't break anything, probably not a bad idea no matter how the rest of the ticket goes?

Reply:
This makes me itchy, but I like the idea.

You _could_ write your own json format for `Submission`; we have custom formatters in many places.

Alternately - would you consider breaking the `SubmissionOptions` refactor out into a separate PR? That would allow it to be reviewed separately and merge earlier to get it some burn-in.